decidim-proposals/app/queries/decidim/proposals/similar_proposals.rb

```diff
@@ -2,14 +2,14 @@
 
 module Decidim
   module Proposals
-    # Class uses to retrieve similar proposals.
+    # Class used to retrieve similar proposals.
     class SimilarProposals < Rectify::Query
       include Decidim::TranslationsHelper
 
       # Syntactic sugar to initialize the class and return the queried objects.
       #
       # features - Decidim::CurrentFeature
-      # form - Decidim::Proposals::ProposalWizardForm
+      # form - Decidim::Proposals::Proposal
```

Comment:
Wrong docs?

Reply:
No this is correct, maybe I could rename `form` to `proposal` as is the object used.
I'll fix Line 20 which is incorrect :disappointed: